homeassistant/components/modbus/sensor.py

```diff
@@ -153,6 +153,8 @@ async def async_setup_platform(
             )
             continue
 
+        regSizeSwap = entry.get(CONF_REGISTER_SIZE, 2) / entry[CONF_COUNT]
```

Comment:
Now if you made the variable earlier you could use it in line 126. The name is strange there are no special register size for swap.